api/services/ocean/gke/paths/clusterCosts.yaml

```diff
@@ -12,6 +12,14 @@ get:
   tags:
     - "Ocean GKE"
   parameters:
+    - in: path
```

Comment:
@vlad666spot dont we have a ready file with reference for this?

Reply:
@sagishoshan , we have: OCEAN_ID and oceanClusterId . They do not correspond to out route 

https://api.spotinst.io/mcs/kubernetes/cluster/{clusterId}/costs (as you can see from here we need path param with exact name - clusterId ) 
![Screenshot 2020-11-10 at 11 57 09](https://user-images.githubusercontent.com/64230439/98658975-55dd8680-234c-11eb-8828-d54113685f02.png)